hs_core/templates/pages/my-resources.html

```diff
@@ -96,23 +96,12 @@ <h4 class="panel-title">
                                     <div class="dropdown-menu dropdown-menu-left shadow-md" role="menu">
                                         <form class="form-horizontal" role="form">
                                             <div class="form-group">
-                                                <label for="filter">Resource Type</label>
+                                                <label for="filter">Type</label>
                                                 <select id="input-resource-type" class="form-control pull-left">
                                                     <option value="" selected>All</option>
                                                     <option value="collection resource">Collection</option>
-                                                    <option value="composite resource">Composite Resource</option>
-                                                    <option value="generic">Generic</option>
-                                                    <option value="geographic feature">Geographic Feature (ESRI Shapefiles)</option>
-                                                    <option value="geographic raster">Geographic Raster</option>
-                                                    <option value="his referenced time series">HIS Referenced Time Series</option>
-                                                    <option value="model instance resource">Model Instance</option>
-                                                    <option value="model program">Model Program</option>
-                                                    <option value="modflow model instance resource">MODFLOW Model Instance</option>
-                                                    <option value="multidimensional">Multidimensional (NetCDF)</option>
-                                                    <option value="script resource">Script Resource</option>
```

Comment:
do we need to update the resource types here:
https://github.com/hydroshare/hydroshare/blob/4853-resource-type-name-fix/theme/templates/pages/group-unauthenticated.html#L189

Reply:
Yeah. Thanks for catching this.